doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -154,12 +154,25 @@ weaknesses in Kubernetes clusters and
 whether Kubernetes is deployed securely by running the checks documented in the
 [CIS Kubernetes Benchmark](https://www.cisecurity.org/benchmark/kubernetes/).
 
-At the time of writing, no clear security rules are defined as mandatory
-(e.g. sec.std.001: The Cloud Operator **should** comply with Center for
-Internet Security CIS Controls) which would have asked for an update of the
-default
-[Functest Kubernetes Security](https://git.opnfv.org/functest-kubernetes/tree/docker/security/testcases.yaml?h=stable%2Fkali)
-behavior (all failures and warnings are only printed).
+[kube-hunter](https://github.com/aquasecurity/kube-hunter) classifies all
+vulnerabilities as low, medium, and high. In context of this conformance suite,
+only the high vulnerabilities lead to a test case failure. Then all low and
+medium vulnerabilities are only printed for information.
+
+Here are the
+[vulnerability categories](https://github.com/aquasecurity/kube-hunter/blob/v0.3.1/kube_hunter/core/events/types.py)
+tagged as high by
+[kube-hunter](https://github.com/aquasecurity/kube-hunter):
+- RemoteCodeExec
+- IdentityTheft
+- PrivilegeEscalation
+
+At the time of writing, no clear Center for Internet Security (CIS) rules are
+defined as mandatory (e.g. sec.std.001: The Cloud Operator **should** comply
+with Center for Internet Security CIS Controls) which would have asked for an
+update of the default kube-bench behavior (all failures and warnings are only
+printed) as integrated in
```

Comment:
This basically says that due to sec.std.001 being a SHOULD requirement, the kube-bench tests actually should not be included in RC2 which only covers MUST requirements.

Should we add a sentence that the kube-bench test suite is "informational only" for the Baraque release, i.e., not a mandatory component of RC2 until the exception is in place?

Reply:
@gkunz @tomkivlin sec.std.001 is about CIS checks as we are discussing in the related issue.
Here it's about the highest known CVE (e.g. PrivilegeEscalation). I would strongly rather consider a missing CNTT requirement than skipping it in RC2.

Reply:
@gkunz 
I would consider this tests is covered by sec.gen.008 All servers part of Cloud Infrastructure **must** be regularly updated to address security vulnerabilities. Do you agree here?

It would have been a little bit more precised to mention the security vulnerability types in req as in RC2.

Also extracted from ref_model/chapters/chapter07.md
Components should be analysed: mechanisms to validate components of the platform stack by checking libraries and supporting code against the Common Vulnerabilities and Exposures (CVE) databases to determine whether the code contains any known vulnerabilities must be embedded into the NFVI architecture itself.  Some of the components required include tools for checking common libraries against CVE databases integrated into the deployment and orchestration pipelines.

Reply:
@collivier  Well, yes and no. I fully support the inclusion of kube-bench as stated elsewhere previously.

I am just trying to highlight that we are not fully following the testing and traceability principles here - and the text is even stating that there are no mandatory requirements as of the time of writing.

So, _including_ (not skipping) kube-bench as "informational" test would be inline with sec.std.001 being a SHOULD requirement but still allowing to leverage the functionality of Functest as is.

Regarding the requirements: sec.std.001 is the right requirement for this CIS Benchmark testsuite. IMHO sec.gen.008 "All servers part of Cloud Infrastructure **must** be regularly updated to address security vulnerabilities." is actually not testable by RC-2 at all as it talks about an _update_.